pages/1.8/version-policy/index.md

```diff
@@ -2,8 +2,8 @@
 layout: layout.pug
 navigationTitle:  Mesosphere Versioning and Release Policy
 title: Mesosphere Versioning and Release Policy
-menuWeight: 5
-excerpt:
+menuWeight: 10
+excerpt: Understanding the format of Mesosphere DC/OS versioning
```

Comment:
This I might change to "Understanding Mesosphere's software versioning and lifecycle policies."